src/state/lottery/helpers.ts

```diff
@@ -143,7 +143,7 @@ export const processRawTicketsResponse = (ticketsResponse: UserTicketsResponse):
 
 export const getViewUserTicketInfoCalls = (totalTicketsToRequest: number, account: string, lotteryId: string) => {
   let cursor = 0
-  const perRequestLimit = 1000
+  const perRequestLimit = 100
```

Comment:
Increased margin for success and reduce frequency of 'out of gas' error when requesting tickets with multicall.

Assume a ticket request will error when requesting 4600 tickets, and a user has 4500 tickets:

If requesting 1000 tickets per call, then the request for 4000 - 5000 will fail. Whereas when only requesting 100 per call, 4400 - 4500 will succeed.